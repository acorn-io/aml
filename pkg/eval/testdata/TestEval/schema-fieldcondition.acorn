schema {
    compare: number <= 0 || number < 1 || number > 3 || number >= 1
    union: number > 1 || {
        hi: 1
        first: true
    } || {
        hi: "bye"
        second: true
    }
}

{
		compare: 1
		union: {
				hi: "str"
		}
}
