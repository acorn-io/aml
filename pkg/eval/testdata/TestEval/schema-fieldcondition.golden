{
  "compare": 1,
  "union": {
    "hi": "str",
    "second": true
  }
}